src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Roslyn/Microsoft.DotNet.Scaffolding.Roslyn.csproj

```diff
@@ -1,7 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net8.0</TargetFrameworks>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Shouldn't need ImplicitUsings or Nullable in the .csproj file since they're set in various D.B.props files now

Reply:
yepyep, will remove